### PR TITLE
Refactored inline styling to classes [Fixes #261]

### LINKED ISSUE #261: Extract inline styling to CSS classes
There are many stray uses of inline styling left over from initial templating. We should clean these up and extract them into CSS classes. The most notable file would be `RoundInformation.vue` but there are others that would benefit:

image

### PR DESCRIPTION
**Pull request by wackerow**
- Extracts all instances of inline styling to CSS classes.
- Added a few recurrent single-purpose classes to `_theme.scss`
- Exceptions include dynamic styling that depends on props being passed to inline `style` attribute (ie: `IconStatus.vue` and `ProgressBar.vue`)

### CODE REVIEW

**Review by corwintines — APPROVED** [review]

File: vue-app/src/views/VerifyLanding.vue [hunk 1]
```diff
@@ -52,7 +52,7 @@
         The current round is no longer accepting new contributions. You can
         still get BrightID verified to prepare for next time.
       </div>
-      <div class="btn-container">
```
Comment (samajammin) [inline vue-app/src/views/VerifyLanding.vue:55]: I think we still want this `btn-container` class.

Current `develop` preview:
clr fund - clr fund 2021-08-26 at 1 31 35 PM

This branch's preview:
clr fund - clr fund 2021-08-26 at 1 31 59 PM

**Review by samajammin — CHANGES_REQUESTED** [review]
Looks great for the most part! 💪 

See one comment - just some styling issues with the button layout on the verify landing page.

File: vue-app/src/views/VerifyLanding.vue (hunk 1 above)
Comment (wackerow) [reply vue-app/src/views/VerifyLanding.vue:55]: Good catch 👍🏻

**Comment by wackerow** [comment]
Fixed the button container, should be good now

**Review by samajammin — APPROVED** [review]

**Pull request merged**